packages/components/src/theme/ComposerTheme/Components/Symbols.ts

```diff
@@ -0,0 +1,17 @@
+import { ComponentStyleConfig } from '@chakra-ui/react';
+
+export const Symbols: ComponentStyleConfig = {
```

Comment:
Please check again the tokens figma file, there was an error and all of the tokens have been updated.

Reply:
Fixed